apps/dashboard/pkg/migration/migrate_test.go

```diff
@@ -69,20 +69,18 @@ func TestMigrate(t *testing.T) {
 			require.JSONEq(t, string(expectedDash), string(outBytes), "%s input check did not match", f.Name())
 		})
 
-		for targetVersion := inputVersion + 1; targetVersion <= schemaversion.LATEST_VERSION; targetVersion++ {
-			testName := fmt.Sprintf("%s v%d to v%d", name, inputVersion, targetVersion)
-			t.Run(testName, func(t *testing.T) {
-				testMigration(t, inputDash, name, inputVersion, targetVersion)
-			})
-		}
+		testName := fmt.Sprintf("%s v%d to v%d", name, inputVersion, schemaversion.LATEST_VERSION)
+		t.Run(testName, func(t *testing.T) {
+			testMigration(t, inputDash, name, inputVersion, schemaversion.LATEST_VERSION)
+		})
```

Comment:
Here is where we just generate one output once the all the migrations on the input JSON are completed